os/tools/mkbinheader.py

```diff
@@ -102,10 +108,65 @@ def get_static_ram_size(bin_type):
     os.remove(STATIC_RAM_ESTIMATION)
     return static_ram_size
 
+############################################################################
+#
+# Kernel binary header information :
+#
+# total header size is (16 + #)bytes.
+# +-----------------------------------------------------------------------------------
```

Comment:
where is crc?

Reply:
CRC is generated by mkchecksum.py for whole binary including binary header after this script is executed.